application/classes/Ushahidi/Core.php

```diff
@@ -162,8 +167,11 @@ public static function init()
 
 		// Post Exporter
 		$di->setter['Ushahidi\Console\Application']['injectCommands'][] = $di->lazyNew('Ushahidi_Console_PostExporter');
+		$di->setter['Ushahidi_Console_PostExporter']['setDatabase'] = $di->lazyGet('kohana.db');
 		$di->setter['Ushahidi_Console_PostExporter']['setPostExportRepo'] = $di->lazyGet('repository.posts_export');
 		$di->setter['Ushahidi_Console_PostExporter']['setDataFactory'] = $di->lazyGet('factory.data');
+		$di->setter['Ushahidi_Console_PostExporter']['setPostDataExportRepo'] = $di->lazyGet('repository.postdataexport');
```

Comment:
I think the current problem is that repository.posts_export is no longer an export, but a FilteredPostsCollection  (or, whatever,  I can think of a new and improved name in a bit if this makes some sense) 

The problem with what I'm thinking above is that we already have collections/sets so this makes less sense in that context, it creates more confusion. I'm starting to wonder if we can't instead of post data export use the saved searches system for this by having an 'exportable' flag, but that might be a bit of a stretch, and the requirements are not the same because you don't have to sort and get headers in sets, for instance. I think eventually, tho, this might be the direction to clean things up even more . 
`TLDR: This is an export, this makes sense as an export. But the posts export we HAD, that one is no longer an export. Does that make sense? ` 
I'm still thinking about this, just trying to capture my thoughts and posting here because you might have a better idea based on this rant